### DISCUSSION TITLE
Guess 🎥🍿

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1085] [Galaxy Trivia] [answered]
**SEVENTEENTH COW FORSOOK CLUE**
**HIS DRILL SCENTS**
**SHEEN HATTED WORKMANSHIP**
**UNHEALTHY HOTBED GO GET DAD**

**Comment by peterbaccaro** [comment, marked as answer]
1. **One Flew Over the Cuckoo's Nest**
2. **Schindler's list**
3. **The Shawshank Redemption**
4. **The Good, The Bad and The Ugly**
